modules/ROOT/pages/database-internals/store-formats.adoc

```diff
@@ -1,33 +1,106 @@
-This allows more related data to be fetched by fewer read operations, resulting in better resource utilization.
-
-The older store formats are based on the original `standard` format, each with different features and limitations.
-`aligned`, `standard`, and `high_limit` formats store most graph data in linked list-like structures on disk.
-The `aligned` format performs better than the `standard` but requires slightly more disk space.
-The `high-limit` format allows more nodes and relationships to be stored than `standard` but performs slightly worse and requires more disk space.
+Store formats are responsible for how data is written to disk and how it is read.
+The choice of store format can affect the performance and resource consumption of read and write transactions, the size of database files on disk, and the amount of data that can be stored.
+
+// * Which files/filenames exist in the database directory.
+// * Performance in memory-constrained environments.
+// The same query on different formats may have a different page cache hit ratio when the database does not entirely fit in the page cache.
+// * The order of query results when the order is not specified.
+// Different store formats may traverse and return data in different order.
+// * Algorithms used by the Neo4j Admin tools.
+// Especially `neo4j-admin database check` and `neo4j-admin database import` commands.
+// Performance and resource utilization may differ.
+
+[[store-format-overview]]
+== Available store formats
+
+Here’s an overview of the available formats and their features:
+
+Block:: label:enterprise-edition[] label:version-number[GA in Neo4j 5.16] +
+* Default format from Neo4j 5.22 for new databases (unless `db.format` is specified).
+* Best performance, supports the highest limits, uses advanced data structures and inlining techniques for better data locality.
+* Allows more related data to be fetched by fewer read operations, enhancing resource utilization.
+* Able to run graphs at large scales.
+* For store format version compatibility and limits, see <<block-format, Block format>>.
+
+Aligned::
+* Default format in Community Edition and for all new databases in Enterprise Edition prior to Neo4j 5.22.
+* Based on `standard`.
+* Performs better than `standard` but requires slightly more disk space.
+* Stores graph data in linked list-like structures on disk.
+* For store format version compatibility and limits, see <<aligned-format, Aligned format>>.
+
+Standard:: label:deprecated[Deprecated in 5.23]
+* Basic, foundational format.
+* Stores graph data in linked list-like structures on disk.
+* For store format version compatibility and limits, see <<standard-format, Standard format>>.
+
+High_limit:: label:enterprise-edition[] label:deprecated[Deprecated in 5.23]
+* Based on `standard` but allows more nodes and relationships.
+* Performs slightly worse than `standard` and requires more disk space.
```

Comment:
worse than `standard`? really?

Reply:
Yes, because it is designed for large databases.